twoopstracker/authentication/urls.py

```diff
@@ -1,10 +1,18 @@
+from dj_rest_auth.registration.views import ConfirmEmailView, VerifyEmailView
 from django.conf.urls import include
 from django.urls import path
 
 from .views import GoogleLogin
 
 urlpatterns = [
+    path("verify-email/", VerifyEmailView.as_view(), name="rest_verify_email"),
```

Comment:
Isn't this part of `registration/` below?

Reply:
@kilemensi yes, you're right!